src/gatsby-plugin-theme-ui/index.js

```diff
@@ -51,7 +52,15 @@ export default {
   },
   buttons: {
     primary: {
+      borderWidth: 3,
       borderRadius: 10,
+      borderColor: 'text',
+      borderStyle: 'solid',
+      backgroundColor: 'secondary',
+      px: 3,
+      py: 2,
+      color: 'white',
```

Comment:
This should probably be `text`? (cc @vivieli77)

Reply:
nah the button is the text color for some reason